tests/test_hosting_commands.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from syncano_cli.hosting.utils import HostingCommands
+from syncano_cli.main import cli
+from tests.base import BaseCLITest
+
+
+class HostingCommandsTest(BaseCLITest):
```

Comment:
Does it create a new instance? Or is the instance common for all tests and gets cleaned up afterwards?